src/Search/SearchHandler.php

```diff
@@ -59,15 +58,11 @@ public function __construct($deprecatedPoolOrCaseSensitive, $caseSensitive = tru
     }
 
     /**
-     * @param string $term
-     * @param int    $page
-     * @param int    $offset
-     *
      * @throws \RuntimeException
      *
      * @return PagerInterface|false
```

Comment:
Im on phone so I cant look at the impact ; but it could be the right time to change the return type to PagerInterface|null